reactor-netty-http/src/main/java/reactor/netty/http/server/HttpServerMetricsRecorder.java

```diff
@@ -54,4 +54,14 @@
 	 * @param time the total time in nanoseconds for the request/response
 	 */
 	void recordResponseTime(String uri, String method, String status, Duration time);
+
+	/**
+	 * Updates the number of active in-use http connections
+	 *
+	 * @param uri the requested URI
+	 * @param method the HTTP method
+	 * @param amount the number of connections to add in the meter
+	 * @since 1.0.15
+	 */
+	default void incrementActiveConnections(String uri, String method, int amount) { }
```

Comment:
same as in tcp, split into 2 methods

```java
recordConnectionActive(String uri, String method);
recordConnectionInactive(String uri, String method);
```

regarding `uri`, as stated in previous review, this seems to be more of a constant `http` closer to a protocol. I don't think we need/want to keep track of number of active connections per uri. Tagging the method might make sense (ie. look up how many active PUT requests are being served).

Reply:
disregard that comment about `uri`. For active connections, it is indeed correctly passed (and the `tag mapper` can be used to reduce the cardinality) 👍